invokeai/frontend/web/src/features/controlLayers/konva/CanvasTool/CanvasMoveToolModule.ts

```diff
@@ -20,12 +21,34 @@ export class CanvasMoveToolModule extends CanvasModuleBase {
     this.manager = this.parent.manager;
     this.path = this.manager.buildPath(this);
     this.log = this.manager.buildLogger(this);
-
     this.log.debug('Creating module');
   }
 
   /**
    * This is a noop. Entity transformers handle cursor style when the move tool is active.
    */
   syncCursorStyle = noop;
+
+  onKeyDown = (e: KeyboardEvent) => {
+    // Support moving via arrow keys
+    const OFFSET = 1; // How much to move, in px
+    const offsets: Record<string, { x: number; y: number }> = {
+      ArrowLeft: { x: -OFFSET, y: 0 },
+      ArrowRight: { x: OFFSET, y: 0 },
+      ArrowUp: { x: 0, y: -OFFSET },
+      ArrowDown: { x: 0, y: OFFSET },
+    };
+    const { key } = e;
+    const selectedEntity = this.manager.stateApi.getSelectedEntityAdapter();
+    const { x: offsetX = 0, y: offsetY = 0 } = offsets[key] || {};
+
+    if (
+      !(selectedEntity && selectedEntity.$isInteractable.get() && $focusedRegion.get() === 'canvas') ||
```

Comment:
Getting the selected entity is the most expensive part check, so we can be a bit more verbose to improve perf when checking if we should bail early. For example:

```ts
if ($focusedRegion.get() !== 'canvas') {
  return;
}
const offsets = KEY_TO_OFFSETS[e.key];
if (!offsets) {
  return;
}
const selectedEntity = this.manager.stateApi.getSelectedEntityAdapter();
if (!selectedEntity || !selectedEntity.$isInteractable.get()) {
  return;
}
```

Also, I forgot to mention we should also bail if `this.manager.$isBusy.get() === true`. This flag indicates that something else is happening on the canvas.